# Can't name it `external_data_bazel` due to Python package clashes.
# @ref https://github.com/bazelbuild/bazel/issues/3998
workspace(name = "external_data_bazel_pkg")
